packages/dui3/lib/common/generated/gql/graphql.ts

```diff
@@ -1913,11 +1913,8 @@ export type OnboardingCompletionInput = {
 };
 
 export enum PaidWorkspacePlans {
-  Business = 'business',
-  Plus = 'plus',
   Pro = 'pro',
```

Comment:
Backwards incompatible GQL API change, hopefully we're fine with that (we can't remove old constants without doing this)